config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     config.cache_max_age = ENV['CACHE_MAX_AGE'] || 10.seconds
 
     config.middleware.use Rack::BounceFavicon
+    require 'bounce_browserconfig'
```

Comment:
Ahh OK yeh. You could use `config.middleware.insert_before` from an initialiser?